src/Configuration.h

```diff
@@ -62,6 +60,7 @@ class Configuration
 
   void setCANID(byte canid);
   void setModuleMode(ModuleMode m);
+  void setHeartbeat(bool beat);
```

Comment:
Should we have a similar function for event ack? Or is this something that will come later?

Reply:
If event ack is to be persisted then yes, we should.  I wasn't going to put it in yet until the code is written for the function in configuration.cpp.  Mind you, that could probably be done now, as well!
I think that this flag should be in eeprom(4) as it is a persisted service specific.